### PR TITLE
Fixes #100 Add CONTRIBUTING.md 

### LINKED ISSUE #100: Add CONTRIBUTING.md
**Actual Behaviour**

Currently, no CONTRIBUTING GUIDE is present in `.github` DIR which is required to follow best practices.

**Expected Behaviour**

CONTRIBUTING.md should be present in `.github` 

**Steps to reproduce it**

Add steps to reproduce bugs or add information on the place where the feature should be implemented. Add links to a sample deployment or code.

**LogCat for the issue**

Provide logs for the crash here

**Screenshots of the issue**

Where-ever possible attach a screenshot of the issue.

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #100 

#### By submitting this pull request I confirm I've read and complied with the below declarations.
- [x] I have tested the changes locally and they are functional.
- [ ] I ensure that I sent this pull request after being assigned to the corresponding issue.
- [x] This pull request has a descriptive title. For example, `Added/Fixed {Feature/Bug}`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

@BrainBuzzer @aashutoshrathi Please review. Thanks!!

### CODE REVIEW

**Comment by BrainBuzzer** [comment]
Hey. Thanks for your contribution. Would you please squash those commits?

**Comment by bhaveshAn** [comment]
Done @BrainBuzzer

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Contributions Best Practices
+
+**Commits**
+* Write clear meaningful git commit messages (Do read http://chris.beams.io/posts/git-commit/)
+* Make sure your PR's description contains GitHub's special keyword references that automatically close the related issue when the PR is merged. (More info at https://github.com/blog/1506-closing-issues-via-pull-requests )
+* When you make very very minor changes to a PR of yours (like for example fixing a failing travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
+* When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
+
+**Code Styleguide**
+* Do follow the .editorconfig file regarding maintaining of code style (It's mandatory).
```
Comment (aashutoshrathi) [inline .github/CONTRIBUTING.md]: Give link to `.editorconfig` of the repo.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Contributions Best Practices
+
+**Commits**
+* Write clear meaningful git commit messages (Do read http://chris.beams.io/posts/git-commit/)
```
Comment (aashutoshrathi) [inline .github/CONTRIBUTING.md]: Make it 

> * Write clear and meaningful commit messages

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+**Code Styleguide**
+* Do follow the .editorconfig file regarding maintaining of code style (It's mandatory).
+* For more information regarding .editorconfig file, see [editorconfig](http://editorconfig.org/#download)
+
+**Feature Requests and Bug Reports**
+* When you file a feature request or when you are submitting a bug report to the [issue tracker](https://github.com/fossasia/susi_desktop/issues), make sure you add steps to reproduce it. Especially if that bug is some weird/rare one.
+
+**Join the development**
+* Before you join development, please set up the project on your local machine, run it and go through the application completely. Press on any button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
```
Comment (aashutoshrathi) [inline .github/CONTRIBUTING.md]: Better not to use emojis in Documentation.

**Comment by bhaveshAn** [comment]
@aashutoshrathi You can review now. Suggested changes done.

**Pull request merged**